pkg/storaged/stratis-details.jsx

```diff
@@ -55,6 +56,17 @@ import { std_reply, with_keydesc, with_stored_passphrase, confirm_tang_trust, ge
 
 const _ = cockpit.gettext;
 
+export function check_stratis_warnings(client, enter_warning) {
+    if (!client.features.stratis_grow_blockdevs)
+        return;
+
+    for (const p in client.stratis_pools) {
+        const blockdevs = client.stratis_pool_blockdevs[p] || [];
+        if (blockdevs.some(bd => bd.NewPhysicalSize[0] && Number(bd.NewPhysicalSize[1]) > Number(bd.TotalPhysicalSize)))
+            enter_warning(p, { warning: "unused-blockdevs" });
```

Comment:
What's that magic string? It's not in main, and it doesn't appear anywhere else in this patch. Do you perhaps mean "unused-space"?

Reply:
The exact string doesn't matter, this is just to get a warning triangle next to the pool in the "Devices" panel on the Overview.

Reply:
Oh, and we don't test that it appears...